clang/lib/Sema/SemaRISCVVectorLookup.cpp

```diff
@@ -463,7 +464,7 @@ void RISCVIntrinsicManagerImpl::CreateRVVIntrinsicDecl(LookupResult &LR,
 bool RISCVIntrinsicManagerImpl::CreateIntrinsicIfFound(LookupResult &LR,
                                                        IdentifierInfo *II,
                                                        Preprocessor &PP) {
-  StringRef Name = II->getName();
+  StringRef Name = II->getName().substr(8);
```

Comment:
What's the magic number of 8?

Reply:
Looks like it is the string length of `__riscv_`.

Reply:
Add an assert that the string starts with `__riscv`

Reply:
Can we use `drop_front(8)`. I find that easier to read than substr with one argument.

Reply:
We can't add an assertion since other tokens go through this function too, or can we filter out the token?

Reply:
I don't follow. If the name can start with things other than "__riscv_" why is the substr(8) ok?

Reply:
Or are you assuming that if the string doesn't start with "__riscv" then we'll produce a garbage string that won't match something in `OverloadIntrinsics`?

Reply:
Yes, if it doesn't match, it won't create the intrinsic.

Reply:
Will this cause something like `XXXXXXX_vadd_vv` to be recognized as a valid RISC-V intrinsic name?

Reply:
Yes, I think we should early return if `consume_front` return `false`.